libcore/src/lib.rs

```diff
@@ -134,7 +134,7 @@ pub fn prepare_ecdsa_signature(
 }
 
 /// Encodes the provided public key and signature into a vector of field elements in the format
-/// expected by `stdlib::crypto::dsa::ecdsa::secp256k1::verify_ecdsa_k256_keccak` procedure.
+/// expected by `core::crypto::dsa::ecdsa_secp256k1::verify_ecdsa_k256_keccak` procedure.
```

Comment:
I think the procedure here should be `miden::core::crypto::dsa::ecdsa_k256_keccak::verify` (also relevant in a couple other places in this file).

Reply:
Similar to previous comment, since there's both a `verify_ecdsa_k256_keccak` and `verify` procedure, I assume that this comment really meant the `verify_ecdsa_k256_keccak` procedure. cc @adr1anh